intermittent_failures.md

```diff
@@ -0,0 +1,95 @@
+# Intermittent Failure Policy
+
+- [Intermittent Failure Policy](#intermittent-failure-policy)
+  - [Introduction](#introduction)
+  - [Policy](#policy)
+    - [Exceptions](#exceptions)
+  - [The Sandbox Suite](#the-sandbox-suite)
+  - [Troubleshooting](#troubleshooting)
+    - [Help! Is my Test Intermittent?](#help-is-my-test-intermittent)
+      - [Step 1: Identify](#step-1-identify)
+      - [Step 2: Confirm](#step-2-confirm)
+      - [Step 3: Document](#step-3-document)
+      - [Step 4: Disable](#step-4-disable)
+  - [Further Reading](#further-reading)
+
+## Introduction
+
+This document describes handling unstable failures which seem to occur randomly. The most likely reason you are reading it is due to an intermittent issue in the [Automated Review Pipeline](https://jenkins.build.o3de.org/job/O3DE/).
+
+The AR pipeline performs builds and tests, as well as additional configuration and asset processing steps. Pull Requests are required to be verified by Automated Review before changes get merged. This Continuous Integration system also executes "branch update jobs" after merges, to detect complex merge-order issues which can occur in the time between verification and merge. A separate pipeline also executes at a slower cadence, which runs a wider array of periodic tests and build variants. [SIG-Build](https://github.com/o3de/sig-build/) owns the Jenkins service and scripts and can advise on build failures, however the tests executed in this pipeline are primarily owned by other SIGs. These tests, along with their applications under test, must operate deterministically.
+
+If you are blocked, you own fixing what blocks you and/or finding support. Even if you are not blocked, the community will appreciate fixes or feedback submitted to the appropriate SIG. Which SIG owns a failure is not always obvious, as the Automated Review pipeline executes code maintained by every SIG in O3DE. If you have questions, reach out over [Discord](https://discord.gg/p3padwr58u)!
+
+## Policy
+
+With few exceptions, no intermittently failing test or feature is allowed. If you see a failure that appears nondeterministic, immediately remove it following these steps:
```

Comment:
Appearances can be deceiving :), modify "If you see a failure that appears nondeterministic" to "If you see a failure that is proven to be nondeterministic"